src/mumble/NetworkConfig.h

```diff
@@ -17,6 +17,8 @@ class NetworkConfig : public ConfigWidget, Ui::NetworkConfig {
 private:
 	Q_OBJECT
 	Q_DISABLE_COPY(NetworkConfig)
+
+
 public:
```

Comment:
This seems like unrelated changes. It will probably removed again by using clang-format though